src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -107,9 +119,196 @@ public static void Profile(string assemblyFileName, string sessionName, Action a
 
             TraceEventSession.MergeInPlace(userFullFileName, Console.Out);
             WriteInfoLine($"ETW Tracing Session saved to \"{userFullFileName}\"");
+            collectOutputFilesCallback(userFullFileName);
+
+            var assemblyModel = GetAssemblyModel(assemblyFileName, userFullFileName, sessionName, performanceTestMessages);
+            var xmlFileName = Path.Combine(outputDirectory, $"{Path.GetFileNameWithoutExtension(userFullFileName)}.xml");
+            WriteXmlFile(xmlFileName, assemblyModel);
+            collectOutputFilesCallback(xmlFileName);
+
+            var mdFileName = Path.Combine(outputDirectory, $"{Path.GetFileNameWithoutExtension(userFullFileName)}.md");
+            DataTable dt = CreateStatistics(assemblyModel);
+            var mdTable = MarkdownHelper.GenerateMarkdownTable(dt);
+            MarkdownHelper.Write(mdFileName, mdTable);
+
+            collectOutputFilesCallback(mdFileName);
+            Console.WriteLine(mdTable);
+        }
+
+        private static DataTable CreateStatistics(AssemblyModel assemblyModel)
+        {
+            var dt = new DataTable();
```

Comment:
Currently this is hardcoded, but I'm planning on fixing this in future PR.

---
In reply to: [101323783](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r101323783) [](ancestors = 101323783)